app/models/ncr/work_order.rb

```diff
@@ -194,6 +201,24 @@ def self.ba80_budget_mailbox
       ENV['NCR_BA80_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com'
     end
 
+    def check_approvers
+      current_approvers = self.approvers.map {|a| a[:email_address]}
+      #remove approving official
+      approving_official = current_approvers.shift
+      if (current_approvers != system_approvers)
+        current_approvers.each do |email|
+          self.remove_approver(email)
+        end
+        system_approvers.each do |email|
+          self.add_approver(email)
+        end
+        approvals = self.approvals
+        if(approvals.first.approved?)
+          approvals.second.make_actionable!
+        end
+      end
+    end
```

Comment:
Can this be combined with `add_approvals` and `update_approver`? It seems there are a lot of ways to set and reset the approval list. I think we can make one entry point that accounts for all combinations?